contracts/src/v0.8/dev/vrf/VRFCoordinatorV2Plus.sol

```diff
@@ -64,6 +64,7 @@ contract VRFCoordinatorV2Plus is VRF, SubscriptionAPI {
   event RandomWordsFulfilled(
     uint256 indexed requestId,
     uint256 outputSeed,
+    uint256 indexed subID,
```

Comment:
Does this need to be indexed?

Reply:
Keep in mind that indexed fields increase the gas cost of emitting the log.

Reply:
Just saw Jin's comment about users using this to track their fulfillments. That makes sense, can ignore this. Just need to keep in mind the gas usage.

Reply:
Sure